R/get_eval_details.R

```diff
@@ -101,8 +101,8 @@ get_eval_details_srv <- function(id, chunks) {
             span()
           ),
           `if`(
-            eval_info_summary["errors"] > 0,
-            get_number_in_box(eval_info_summary["errors"], box_color = "#ff0000", char_color = "#ffffff"),
+            eval_info_summary["quosure.error"] > 0,
```

Comment:
so does this mean the debug info will work?

Reply:
my mistake here when changing text by regular expression. I'll fix it